pkg/model/scalewaymodel/sshkey.go

```diff
@@ -0,0 +1,34 @@
+package scalewaymodel
+
+import (
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/cloudup/scalewaytasks"
+)
+
+// SSHKeyModelBuilder configures SSH objects
+type SSHKeyModelBuilder struct {
+	*ScwModelContext
+	Lifecycle fi.Lifecycle
+}
+
+var _ fi.ModelBuilder = &SSHKeyModelBuilder{}
+
+func (b *SSHKeyModelBuilder) Build(c *fi.ModelBuilderContext) error {
+	name, err := b.SSHKeyName()
+	if err != nil {
+		return err
+	}
+	sshKeyResource := fi.Resource(fi.NewStringResource(string(b.SSHPublicKeys[0])))
+
+	t := &scalewaytasks.SSHKey{
+		Name:      fi.String(name),
+		Lifecycle: b.Lifecycle,
+		PublicKey: &sshKeyResource,
+		//Tags ??
+	}
+	c.AddTask(t)
+
+	// TODO: shouldn't we be able to handle more than 1 ssh key ?
```

Comment:
If the SSH key is passed explicitly from the command line we should only handle this one. I think installing several ssh keys is not in the scope of kops. Let's stick to one ssh key for the time being.